core/domain/feedback_jobs_one_off.py

```diff
@@ -100,3 +102,88 @@ def _cache_last_nonempty_message_author_id(
             thread_model.last_nonempty_message_author_id = message_author_id
             return True
         return False
+
+
+class CleanUpFeedbackAnalyticsModelModelOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """One-off job to remove feedback analytics models for
+    deleted explorations.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [feedback_models.FeedbackAnalyticsModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+        exp_model = exp_models.ExplorationModel.get_by_id(
+            item.id)
+        if exp_model is None or exp_model.deleted:
+            yield ('Deleted Feedback Analytics Model', item.id)
+            item.delete()
+
+    @staticmethod
+    def reduce(key, values):
+        yield (key, values)
+
+
+class CleanUpGeneralFeedbackThreadModelOneOffJob(
```

Comment:
This is not still being done in codebase and instead of running clean ups at regular intervals - this should be fixed via #10809